definitions/marts/tad_marts/tad_npq_enrolment.sqlx

```diff
@@ -78,7 +78,7 @@ SELECT
   employer_name,
   employment_role,
   employment_type,
-  profile_user_id,
+  participant_profile_id as profile_user_id,

```

Comment:
Do the TAD field descriptions also need updating?

Reply:
They don't because everything appears the same to them with these changes.